### PR TITLE
Soen95 admin dashboard

### PR DESCRIPTION
**Pull request by silentswordxv**
# Description

Implements/fixes [SOEN-95](https://soen390team8.atlassian.net/browse/SOEN-95?atlOrigin=eyJpIjoiYjgwYzA3NjI4YzExNDZhN2FjZDE0OTJmMTFjMDdlMjYiLCJwIjoiaiJ9)

- Added Admin Dashboard Navigation
- Added Admin Stats

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Full code coverage via unit tests. Added to appropriate test files.

- backend -> routes/admin.test.js
image

- frontend -> homepage/dashboard-chart.test.js
- frontend -> dashboard.test.js
image
(missing coverage is from placeholder branches + code)
# Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| image | image |

### CODE REVIEW

File: backend/controllers/admin-controller.js [hunk 1]
```diff
@@ -243,10 +243,36 @@ async function rejectAccount(req, res) {
     }
 }
 
+async function getDashboardStats(req, res) {
+    Promise.all([
```
Comment (NicholasYip) [inline backend/controllers/admin-controller.js:247]: good stuff

**Review by NicholasYip — APPROVED** [review]
nice work

**Review by KelvinTrang — APPROVED** [review]
Nicely done chad

**Review by jfv-2000 — APPROVED** [review]
Well done ! Good work

**Pull request merged**